git_clone_test/load_project_test.ipynb

```diff
@@ -1,9 +1,22 @@
 {
```

Comment:
<p>mount_s3 is not safe and seems won't be ever safe.</p><p>you just exposed by yaml print some AWS credentials in this PR, please ask devops to delete exposed AWS_ACCESS_KEY_ID</p><p>and remove the print</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Deleted all non relevant prints</p>